crates/wac-resolver/src/registry.rs

```diff
@@ -164,7 +168,14 @@ impl RegistryPackageResolver {
             if let Some(info) = self
                 .client
                 .registry()
-                .load_package(self.client.get_warg_registry(), &id)
+                .load_package(
+                    self.client
+                        .get_warg_registry(id.namespace())
+                        .await
+                        .map_err(|e| CommandError::WargClient(e))?
```

Comment:
it would be nice if these errors were more context aware. Instead of just storing the `warg` error, we should add some additional context, let that this is fetching the registry.